Automattic/jetpack
# Subscriptions Block: Fix subscriber count incorrect position when padding is enabled.

Fixes #

#### Changes proposed in this Pull Request:

* Fix subscriber count having incorrect position when padding is enabled.

**Technical changes:**
* Add a container `.wp-block-jetpack-subscriptions__container`.
  * This is to allow using flex to the positioning of the subscription count before or after the input fields (for the future, [more explanation here](https://github.com/Automattic/jetpack/pull/24942#issuecomment-1185235336)).
* Standardize subscription count HTML structure to `.wp-block-jetpack-subscriptions__subscount`.

| Before | After |
|---|---|
| ![image](https://user-images.githubusercontent.com/1287077/180780240-b1073e43-0e4c-46bc-8d7d-4e1f8251a3f6.png) | ![image](https://user-images.githubusercontent.com/1287077/180779953-aca4f823-8b1e-448e-91b7-418190fe1eb7.png) |
| ![image](https://user-images.githubusercontent.com/1287077/180780125-53dc3742-a77c-4d17-ab83-bee2a9bd9748.png) | ![image](https://user-images.githubusercontent.com/1287077/180780372-24ddd689-5526-4535-8c00-fcbd0511e495.png) |

| Editor HTML | Frontend HTML  |
|---|---|
|  ![image](https://user-images.githubusercontent.com/1287077/180779812-1f45f4b8-5a34-46e3-8376-4d838a1753cb.png) | ![image](https://user-images.githubusercontent.com/1287077/180779829-10889258-cf31-419a-a953-51279d4fe81f.png) |

#### Other information:


#### Jetpack product discussion
Just fixing a bug.



#### Does this pull request change what data or activity we track or use?
Nope.



#### Testing instructions:






**1. Test right alignment:**
* Make sure you have a block-based theme enabled like Twenty Twenty-Two
* Add **Subscription Block**.
* Under **Block Settings > Settings**, turn on **Show subscription count**.
* Under **Block Settings > Dimensions** add some **padding** to the block.
* Publish the post.
* The count should appear right-aligned to the button.

**2. Test left alignment with newline turned on:**
* Edit the post again.
* Under **Block Settings > Settings**, turn on **Place button on new line**.
* Update the post.
* The count should appear left-aligned to the button.

Test on both the Jurassic Tube (Self-Hosted) version & the WPCOM version (via the Phab patch) as the HTML output is generated differently (as seen on `views.php`).
